engines/e_devcrypto.c

```diff
@@ -1252,9 +1252,7 @@ static int bind_devcrypto(ENGINE *e) {
  * /Richard Levitte, 2017-05-11
  */
 #if 0
```

Comment:
Why not remove the #if 0 block entirely?

Reply:
I'm thinking that if there's a day where /dev/crypto supports this properly, we already have the preparation code in place.